components/footer/index.tsx

```
@@ -41,151 +49,319 @@ export function Footer() {
+        {t("STAY_UPDATED")}
+      </h4>
+      <p className="text-sm text-gray-600 dark:text-gray-400">
+        {t("NEWSLETTER_DESCRIPTION")}
+      </p>
+      <div className="flex gap-2">
+        <input
+          type="email"
+          placeholder={t("ENTER_EMAIL")}
+          className="flex-1 px-4 py-3 rounded-xl bg-white/50 dark:bg-gray-800/50 backdrop-blur-lg border border-white/30 dark:border-gray-700/40 focus:border-blue-300/50 dark:focus:border-blue-500/30 focus:outline-none transition-all duration-300 text-sm placeholder-gray-500 dark:placeholder-gray-400"
+        />
+        <button className="px-6 py-3 rounded-xl bg-gradient-to-r from-blue-500 to-purple-500 hover:from-blue-600 hover:to-purple-600 text-white font-medium transition-all duration-300 hover:shadow-lg hover:shadow-blue-500/25 hover:scale-105 text-sm">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)